changelogs/Spyder-6.md

```diff
@@ -17,7 +17,11 @@
 
 #### Remote Client
 
+* **Breaking** The `create_ipyclient_for_server` and `get_kernels` methods were removed.
+* Add `sig_server_changed` signal to report when a server was added or removed.
+* Add `get_server_name` method to get a server name given its id.
 * Add `register_api` and `get_api` methods in order to get and register new rest API modules for the remote client.
+* Add `get_jupyter_api` method to get the new Jupyter APIO to communicate with the remote server.
```

Comment:
* Please check if this entry is correct.
* Also, what's the difference between `get_api` and `get_jupyter_api`?

Reply:
Looks good to me.